README.md

```diff
@@ -63,7 +63,7 @@ import { ComponentName } from '@carbon/ibm-security';
 const { ComponentName } = require('@carbon/ibm-security');
 ```
 
-[Babel](https://babeljs.io) builds both of these variants and imports `carbon-components-react` in the respective version using a [plugin](https://github.com/carbon-design-system/ibm-security/blob/master/babel/carbon-imports.babel-plugin.js), so that no further transpilation is required.
+[Babel](https://babeljs.io) builds both of these variants and imports `carbon-components-react` in the respective version using a [plugin](https://github.com/carbon-design-system/ibm-security/tree/master/babel/carbon-imports.babel-plugin.js), so that no further transpilation is required.
```

Comment:
Quick question -- just wondering why update this path if it a link to a file, a `blob` (not directory or `tree`), and updates to that path when you go here? 🤔